spec/requests/api/vms_spec.rb

```diff
@@ -620,7 +620,7 @@ def update_raw_power_state(state, *vms)
 
       run_post(vm_url, gen_request(:set_owner, "owner" => "bad_user"))
 
-      expect_single_action_result(:success => false, :message => /.*/, :href => :vm_url)
```

Comment:
Thought this was failing before, but can't seem to reproduce it again. Changing it back :+1: